AVsitter2/Utilities/Missing-anim-finder.lsl

```diff
@@ -64,27 +66,38 @@ default
         {
             if (data == EOF)
             { 
-                if(NOT_FOUND_COUNT>0){
+                if(!NOT_FOUND_COUNT){
+                    Owner_Say("All animations referenced in the notecard were accounted for.");   
+                }
+                else{
                     Owner_Say("Anims were used in the notecard but not found in inventory!");
                 }
                 integer i;
                 for (i = 0; i < llGetInventoryNumber(INVENTORY_ANIMATION); i++)
                 {
+                    integer isVariableSpeed;
                     string anim_basename = llGetInventoryName(INVENTORY_ANIMATION, i);
                     if(llListFindList(["+","-"], [llGetSubString(anim_basename,-1,-1)]) != -1)
                     {
+                        isVariableSpeed=TRUE;
                         anim_basename = llGetSubString(anim_basename,0,-2);
                     }
                     
-                    if (llListFindList(ALL_USED_ANIMATIONS, [anim_basename]) == -1 && llListFindList(["AVhipfix"], [llGetInventoryName(INVENTORY_ANIMATION, i)]) == -1)
+                    integer index = llListFindList(ALL_USED_ANIMS, [anim_basename]);
+                    
+                    if(index != -1 && isVariableSpeed == TRUE && llList2Integer(VARIABLE_SPEED_ANIMS,index) == FALSE){                   
+                        Owner_Say("Variable-Speed Animation '" + llGetInventoryName(INVENTORY_ANIMATION, i) + "' found in inventory but the associated submenu(s) are not set as Variable-Speed submenu(s)!");
+                        UNUSED_ANIMS += llGetInventoryName(INVENTORY_ANIMATION, i);
+                    }
+                    else if (index == -1 && llListFindList(["AVhipfix"], [llGetInventoryName(INVENTORY_ANIMATION, i)]) == -1)
```

Comment:
Why `llListFindList`?

Reply:
```llListFindList(["AVhipfix"], [llGetInventoryName(INVENTORY_ANIMATION, i)]) == -1``` was in there before the patch. The only reason is to enable easy future addition of additional "reserved" animations that should be skipped along with "AVhipfix".

Reply:
It's an easy change to do if the need arises, anyway.